src/plugins/region_map/public/plugin.ts

```diff
@@ -88,16 +91,25 @@ export interface RegionMapsConfig {
 
 export interface RegionMapPluginSetup {
   config: any;
+  addOptionsTab: (options: AddImportMapTab) => void;
```

Comment:
nit: I'd call it `option` in singular because it looks like you can only add a single tab with this call

Reply:
Ohh let me change it as `additionalOptionsTab`.  Is that okay?

Reply:
This should be fine. Its pretty explicit already
```
addOptionsTab: (option: AddImportMapTab) => void;
```